From aa06f15ee329ca772697f9f708853ed0c66c4731 Mon Sep 17 00:00:00 2001
From: DimitriFtres <dimitrifastres@gmail.com>
Date: Sun, 6 Jun 2021 13:19:05 +0200
Subject: [PATCH] Adaptation for internet

---
 BDD/Connexion_bdd.php | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/BDD/Connexion_bdd.php b/BDD/Connexion_bdd.php
index 58bcb00..d52a790 100644
--- a/BDD/Connexion_bdd.php
+++ b/BDD/Connexion_bdd.php
@@ -1,8 +1,8 @@
 <?php
-    $host = "185.98.131.109";
-    $user = "fastr1522145";
-    $password = "pcs33ctjva";
-    $table_name = "fastr1522145";
+    $host = "";
+    $user = "";
+    $password = "";
+    $table_name = "";
 try {
     $bdd = new PDO("mysql:host=$host; dbname=$table_name;charset=UTF8", $user, $password);
 } 
-- 
2.30.0.windows.2

